Zend/tests/ErrorException_getSeverity.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+final public int ErrorException::getSeverity ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - #phparty7 - @phpsp - novatec/2015 - sao paulo - br
+--FILE--
+<?php
+function exception_error_handler($severity) {
+ 
+    $message = "error exception, severity =>";
+    $code = 500;
+    $severity = 75;
+    try{
+        if (!(error_reporting() & $severity)) {
+            return;
+        }
+        throw new ErrorException($message, $code, $severity);
+    } catch(ErrorException $err) {
+        var_dump(
+           $err->getMessage(), 
+           $err->getSeverity(), 
+           $err->getCode(), 
+           $err->getPrevious(), 
+           $err->getFile(), 
+           $err->getLine(), 
+           $err->getTraceAsString()
+        );
+        print(is_array($err->getTrace()) ? ("array();") : ("method fail"));
+    }
+    
+}
+set_error_handler("exception_error_handler");
+/* Trigger exception */
+preg_match();
+?>
+--EXPECTF--
```

Comment:
Most of these values here should be predictable, such as message, severity, code, previous, file & line.